msg/en_US.utf8/cubrid.msg

```diff
@@ -1874,6 +1874,7 @@ $set 8 MSGCAT_SET_PARSER_SEMANTIC
 304 Could not find compatible signature for function %1$s()
 305 Signature %1$s is not compatible with function arguments.
 306 dblink - Required information is missing.
+400 A member of owner group or a member of DBA group can create table.
```

Comment:
It should be 307 in order, so why skip to 400?

Reply:
This is a margin to avoid error number collisions when merging.
However, it seems unnecessary to do this, so I will change the numbers consecutively.